README.md

```diff
@@ -34,74 +34,33 @@ This will start hops in development mode. Visit [http://localhost:8080](http://l
 
 If you want to build the JS bundles for production, run `npm run build` in your terminal and to start a Node.js server just type in `npm start --production`.
 
-### Installation
-
-Besides reasonably recent versions of [Node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com), hops has no global dependencies. If you need those, we recommend using [nvm](https://github.com/creationix/nvm) or similar.
-
-Install the `hops-cli` globally via npm `npm install --global hops-cli` or yarn `yarn global add hops-cli`:
-
-### Project creation
-
-`hops init my-project` will create a directory called `my-project` and initialize an example react project inside it.
-
-### Configuration
```

Comment:
I kinda liked the config intro, because it's an integral part a hops app. Does anything speak against keeping it?

Reply:
I was thinking about that, too, but I came to the conclusion that it is not more integral or important than, say, the CLI stuff.

I’d also like to extend the component tables below with short introductory texts for all of them instead of highlighting a single feature.

Reply:
> I’d also like to extend the component tables below with short introductory texts for all of them instead of highlighting a single feature.

<3